package.json

```diff
@@ -23,7 +23,7 @@
         "prettier": "prettier --ignore-path .gitignore \"**/*.+(ts|tsx|json)\"",
         "cypress:open": "cypress open --env AUTH_MODE=LOCAL",
         "cypress:run": "cypress run --env AUTH_MODE=LOCAL",
-        "precommit": "lint-staged"
+        "precommit": "lerna run --concurrency 1 --stream precommit --since HEAD --exclude-dependents"
```

Comment:
This is... almost working? It works find if there are no lint errors. If there are actual lint errors though it crashes after finding the first error correctly. Crash looks like this

```
cripts: [FAILED] fatal: Unable to create '/Users/worku/Code/MCRRS/managed-care-review/.git/index.lock': File exists.
scripts: [FAILED]
scripts: [FAILED] Another git process seems to be running in this repository, e.g.
scripts: [FAILED] an editor opened by 'git commit'. Please make sure all processes
scripts: [FAILED] are terminated then try again. If it still fails, a git process
scripts: [FAILED] may have crashed in this repository earlier:
scripts: [FAILED] remove the file manually to continue.
scripts:   ✖ lint-staged failed due to a git error.
scripts: [STARTED] Cleaning up temporary files...
````